js/aax.js

```diff
@@ -2037,6 +2038,158 @@ module.exports = class aax extends Exchange {
+        // const notional = this.safeString (position, 'value');
+        const leverage = this.safeString (position, 'leverage');
+        const unrealisedPnl = this.safeString (position, 'unrealisedPnl');
+        const realisedPnl = this.safeString (position, 'realisedPnl');
+        // Initial Position Margin = ( Position Value / Leverage ) + Close Position Fee
+        const takerFee = this.fees.trading.taker;
+        // const feePaid = Precise.stringMul (takerFee, notional);
+        const initialMarginString = this.safeString (position, 'posMargin');
+        const percentage = Precise.stringMul (Precise.stringDiv (unrealisedPnl, initialMarginString), '100');
+        const timestamp = this.safeInteger (position, 'ts');
+        return {
+            'info': position,
+            'symbol': this.safeString (market, 'symbol'),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'initialMargin': this.parseNumber (initialMarginString),
+            // 'initialMarginPercentage': this.parseNumber (Precise.stringDiv (initialMarginString, notional)),
+            // 'maintenanceMargin': this.parseNumber (Precise.stringMul (maintenanceRate, notional)),
+            // 'maintenanceMarginPercentage': this.parseNumber (maintenanceRate),
+            'entryPrice': this.safeNumber (position, 'avgEntryPrice'),
+            // 'notional': this.parseNumber (notional),
+            'leverage': parseInt (leverage),
+            'unrealizedPnl': this.parseNumber (unrealisedPnl),
+            'contracts': this.parseNumber (size),
+            'contractSize': this.safeNumber (market, 'contractSize'),
+            //     realisedPnl: position['realised_pnl'],
+            // 'marginRatio': undefined,
+            'liquidationPrice': this.safeNumber (position, 'liquidationPrice'),
+            'markPrice': this.safeNumber (position, 'marketPrice'),
+            // 'collateral': this.safeNumber (position, 'margin'),
+            'marginType': this.safeString (position, 'settleType'),
+            'side': side,
+            'percentage': this.parseNumber (percentage),
+        };
+    }
+
+    parsePositions (response) {
+        const result = [];
+        const positions = this.safeValue (response, 'data');
+        const ts = this.safeInteger (response, 'ts')
+        for (let i = 0; i < positions.length; i++) {
+            result.push (this.parsePosition (this.extend (positions[i], {'ts': ts})));
+        }
+        return result;
+    }
+
+    async fetchPositions (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (symbols !== undefined) {
+            let symbol = undefined;
+            if (Array.isArray (symbols)) {
+                const symLen = symbols.length;
+                if (symLen > 1) {
+                    throw new BadRequest (this.id + ' fetchPositions symbols argument cannot contain more than 1 symbol');
```

Comment:
I think we should add `fetchPosition` to it, with a required `symbol` argument.

Reply:
It can get multiple positions if you don't pass a symbol

Reply:
Yep, but if you do want to get a single position by a symbol, this exchange can have fetchPosition = true, implemented and unified.